src/css/triggerView.scss

```diff
@@ -288,10 +276,11 @@
     background-color: var(--B100);
     color: var(--B500);
     border-radius: 4px;
-    padding: 2px 8px;
+    padding: 2px 6px;
```

Comment:
suggestion: can completely remove this className and use scss classes for styling

Reply:
This classname is using in multiple components and so avoiding to write everywhere